### DISCUSSION TITLE
Welcome to OpenSumi Discussions!

### DISCUSSION THREAD

**Discussion by Aaaaash** [opensumi/core#252] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tyn1998** [comment]
Hi guys! My name is Lam Tang(唐烨男) and I am currently a graduate student(研一) from [X-lab](https://github.com/X-lab2017) in DaSE, ECNU.

I would like to take part in OSPP Summer 2022 this year and just find this wonderful project on [task list](https://summer-ospp.ac.cn/#/org/orgdetail/dca1f775-3ce8-49f5-82eb-57a1b6082249/)! Tooling and anything that can improve my workflow will drive me crazy and I have a [repo](https://github.com/tyn1998/dotfiles) to manage all configuration files for my tools. I even name the default branch name of the repo as "tIDE", which means terminal based IDE, haha. 

I am fond of this amazing project and will try to run it on my local machine. Keep touch with you guys :)
